tests/server/test_server_validation.py

```diff
@@ -33,21 +32,20 @@ def test_with_validator_json_response(both_fake_remote_clients, capsys):
 
     captured = capsys.readouterr()
     json_response = json.loads(captured.out)
-    assert json_response["failure_count"] == 0
-    assert json_response["internal_failure_count"] == 0
-    assert json_response["optional_failure_count"] == 0
-    assert validator.results.failure_count == 0
-    assert validator.results.internal_failure_count == 0
-    assert validator.results.optional_failure_count == 0
-    assert dataclasses.asdict(validator.results) == json_response
-
+    assert json_response["failure_count"] == 0, json_response
+    assert json_response["internal_failure_count"] == 0, json_response
+    assert json_response["optional_failure_count"] == 0, json_response
+    assert validator.results.failure_count == 0, json_response
+    assert validator.results.internal_failure_count == 0, json_response
+    assert validator.results.optional_failure_count == 0, json_response
+    assert dataclasses.asdict(validator.results) == json_response, json_response
     assert validator.valid
 
 
 def test_mongo_backend_package_used():
```

Comment:
I guess a similar test should be added for the elastic backend, asserting that a variable exists in the globals or is loaded?
Perhaps this will lead to the `client` variable in the elastic.py file to be set to `None` prior to the `if` block and then this can be checked?